src/components/shared/MarkerTooltipContents.js

```diff
@@ -59,7 +59,7 @@ function _markerDetailNullable<T: NotVoidOrNull>(
   value: T | void | null,
   fn: T => string = String
 ): React.Node {
-  if (value === undefined || value === null) {
+  if (value === undefined || value === null || value.length === 0) {
```

Comment:
This line seems a little risky, and I'm not sure what the intent of what exactly it's checking and what type of type will be passed here. I would prefer`(Array.isArray(value) && value.length === 0)` if it's an array check, or `value === ''` for a string check. This will also check numbers and other things, so it's not immediately obvious on the behavior.

Reply:
Thanks you both for the feedback. :)
I went for `value === ''` as this is more readable.

@gregtatum r?